### PR TITLE
[WIP] add support for Schema API for Solr 5

### PR DESCRIPTION
**Pull request by shieldo**
Picking up from PR https://github.com/solariumphp/solarium/pull/346 by @bpolaszek.

Work in progress - have improved coding standards, made some coding tweaks/ fixes, added a test for the schema query class.  Forthcoming: test coverage for everything else.

### CODE REVIEW

**Comment by basdenooijer** [comment]
Nice work!

**Comment by bpolaszek** [comment]
+1024! 
Thanks for taking some time about this. I've been very busy these months, and I'm really happy some motivated guys improved this PR.

File: library/Solarium/Core/Client/Client.php [hunk 1]
```diff
@@ -91,7 +91,12 @@ class Client extends Configurable
     const QUERY_UPDATE = 'update';
 
     /**
-     * Querytype ping.
+     * Querytype schema
```
Comment (thePanz) [inline library/Solarium/Core/Client/Client.php]: Missing a dot (.) in the end of comment, same as in line 99

**Comment by basdenooijer** [comment]
Is this PR considered ready for review, or are there still some known issues? It would be a very nice feature to add!

**Comment by shieldo** [comment]
Sorry, not quite there yet - but this is the reminder I need to get it over the finishing line!

**Comment by bpolaszek** [comment]
*Poke* 

Is there something we can do to get this feature merged? :-)

**Comment by shieldo** [comment]
Poking might help! :) I'm definitely mindful of it.  I'll keep this tab open to remind me.  Would be really great to complete this.

**Comment by guiwoda** [comment]
@shieldo I'm poking too!
Also, any way I can help you? Do you have a list of TODOs or something similar?

**Comment by shieldo** [comment]
@guiwoda Yes, I was just thinking about this! I would need to get my head back in as it's been a few months. Keeping this tab open - I obviously failed at that. I've made a reminder to myself to pick this up at the weekend!

**Comment by guiwoda** [comment]
Awesome. Ping me back then if you need a hand :)

**Pull request closed without merging**